views/js/e2e/_helpers/selectors/setupSelectors.js

```diff
@@ -20,7 +20,7 @@
  * CSS Selectors for setup
  */
 export default {
-    resourceTree: '.resource-tree',
+    resourceTree: '.taotree',
```

Comment:
Use `testsPage: { rootTestClass:'li[data-uri="http://www.tao.lu/Ontologies/TAOTest.rdf#Test"]'`

Reply:
yes, I agree with you. It's much better.